cnf/ext/central.mvn

```diff
@@ -192,3 +192,8 @@ org.slf4j:log4j-over-slf4j:1.7.7
 org.apache.zookeeper:zookeeper:3.4.13
 org.apache.aries.component-dsl:org.apache.aries.component-dsl.component-dsl:1.2.2
 org.apache.aries.typedevent:org.apache.aries.typedevent.bus:0.0.1-SNAPSHOT
+
+org.mockito:mockito-core:jar:3.10.0
```

Comment:
If we are going to add mockito to the build, then we need to add it in a way so that all TCK projects can benefit. 

See https://github.com/osgi/osgi/pull/312 where I added awaitility as a TCK support jar and model adding mockito this way.

Reply:
I extracted versions into junit.bnd